scalableminds/webknossos
# Allow to export current reslices as image files

I've added a shortcut (`q`) for now to trigger the screenshot download. A button would probably be more discoverable. Maybe a small camera icon button at the top level (next to the position?!) or we put it in the dropdown menu :/

Getting an image from a data buffer was harder than I expected (and then webGL's y-axis is inverted to top it off ^^).

### URL of deployed dev instance (used for testing):
- https://screenshot.webknossos.xyz

### Steps to test:
- Press q in any view OR use the screenshot button in the dropdown in any tracing type. One (or more) save prompts should appear to download the currently visible viewport(s) as a png file.
- Names should contain the dataset name, current position and plane. (Add zoom?)
- Node selection should still work.

### Issues:
- fixes #3760

------
